src/aind_data_schema_models/organizations.py

```diff
@@ -258,6 +267,15 @@ class TeledyneFLIR(_Organization):
     registry_identifier: Literal["01j1gwp17"] = "01j1gwp17"
 
 
+class TempletonWorldCharityFoundation(_Organization):
```

Comment:
This needs to be alphabetized properly. I'm not sure why Teledyne FLIR is where it is - I thought we had an issue or a discussion open about this. Regardless, Templeton needs to be in the Ts

Reply:
Oh, I just put Templeton there because of Teledyne. I think FLIR was there because it used to be FLIR, or something of that sort. I will move both.